specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2017-12-01-preview/postgresql.json

```diff
@@ -0,0 +1,1725 @@
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Application-specific metadata in the form of key-value pairs."
+        }
+      },
+      "description": "Parameters allowd to update for a server."
+    },
+    "ServerListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Server"
+          },
+          "description": "The list of servers"
+        }
+      },
+      "description": "A list of servers."
+    },
+    "FirewallRuleProperties": {
+      "properties": {
+        "startIpAddress": {
+          "type": "string",
+          "pattern": "^(([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])\\.){3}([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])$",
+          "description": "The start IP address of the server firewall rule. Must be IPv4 format."
+        },
+        "endIpAddress": {
+          "type": "string",
+          "pattern": "^(([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])\\.){3}([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])$",
+          "description": "The end IP address of the server firewall rule. Must be IPv4 format."
+        }
+      },
+      "required": [
+        "startIpAddress",
+        "endIpAddress"
+      ],
+      "description": "The properties of a server firewall rule."
+    },
+    "FirewallRule": {
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/FirewallRuleProperties",
+          "x-ms-client-flatten": true,
+          "description": "The properties of a firewall rule."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
+        }
+      ],
+      "required": [
+        "properties"
+      ],
+      "description": "Represents a server firewall rule."
+    },
+    "FirewallRuleListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/FirewallRule"
+          },
+          "description": "The list of firewall rules in a server."
+        }
+      },
+      "description": "A list of firewall rules."
+    },
+    "DatabaseProperties": {
+      "properties": {
+        "charset": {
+          "type": "string",
+          "description": "The charset of the database."
+        },
+        "collation": {
+          "type": "string",
+          "description": "The collation of the database."
+        }
+      },
+      "description": "The properties of a database."
+    },
+    "Database": {
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/DatabaseProperties",
+          "x-ms-client-flatten": true,
+          "description": "The properties of a database."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
```

Comment:
Is database really a proxy resource? Databases under Microsoft.SQL is a tracked resource. So it has location, tags properties also.

Reply:
rdbms only has servers as tracked resource. It only has "database" as conceptually and we only having them as proxy resource